static/js/publisher/release/actions/modal.js

```diff
@@ -0,0 +1,15 @@
+export const OPEN_MODAL = "OPEN_MODAL";
+export const CLOSE_MODAL = "CLOSE_MODAL";
+
+export function openModal(payload) {
+  return {
+    type: OPEN_MODAL,
+    payload: { payload }
```

Comment:
This causes nested `payload` object, so action looks like: `{ type: "OPEN", payload: { payload: { ... } }` and as a result reducer also contains `{ isOpen: true, payload: { ... }}` when it should probably contain modal props directly.